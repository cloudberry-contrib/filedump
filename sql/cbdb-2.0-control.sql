-- control file
\! pg_filedump -c test/pg_control.cbdb-2.0
