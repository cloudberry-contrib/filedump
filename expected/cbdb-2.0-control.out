-- control file
\! pg_filedump -c test/pg_control.cbdb-2.0

*******************************************************************
* PostgreSQL File/Block Formatted Dump Utility
*
* File: test/pg_control.cbdb-2.0
* Options used: -c
*******************************************************************

<pg_control Contents> *********************************************

                          CRC: Correct
           pg_control Version: 13000700
              Catalog Version: 302502091
            System Identifier: 7490468541381486304
                        State: IN PRODUCTION
                Last Mod Time: Tue Apr  8 07:48:43 2025
       Last Checkpoint Record: Log File (0) Offset (0x0d0189f8)
  Last Checkpoint Record Redo: Log File (0) Offset (0x0d0189c0)
          |-       TimeLineID: 1
          |-         Next XID: 0/1200
          |-         Next OID: 27331
          |- Next Relfilenode: 24818
          |-       Next Multi: 1
          |-    Next MultiOff: 0
          |-             Time: Tue Apr  8 07:48:43 2025
       Minimum Recovery Point: Log File (0) Offset (0x00000000)
          Backup Start Record: Log File (0) Offset (0x00000000)
            Backup End Record: Log File (0) Offset (0x00000000)
End-of-Backup Record Required: no
       Maximum Data Alignment: 8
        Floating-Point Sample: 1234567
          Database Block Size: 32768
           Blocks Per Segment: 32768
              XLOG Block Size: 32768
            XLOG Segment Size: 67108864
    Maximum Identifier Length: 64
           Maximum Index Keys: 32
             TOAST Chunk Size: 8140

